### PR TITLE
Improve README to provide a link to a Bazel example.

### PR DESCRIPTION
**Pull request by myronmarston**
This resolves one aspect of #2066.

### CODE REVIEW

File: README.md
Comment (sandwwraith) [inline README.md:342]: Are you sure that fixing `v1.6.0` tag is a good idea? E.g. there is an update in `master` that removes kotlin-reflect from test dependencies.

**Review by sandwwraith — CHANGES_REQUESTED** [review]

File: README.md
Comment (myronmarston) [reply README.md:342]: As a general rule, I _always_ include a SHA or tag in a github code link rather than linking to a file on the master or main branch. Otherwise, the link might stop working at some point.  For example, if the maintainers of that repo reorganize the directory structure at some point such that in `master` a file or directory at that path no longer exists...then the link would become a 404.

Also, if someone follows this link and wants to see the latest version of the directory, it's easy for them to replace `v1.6.0` with `master` in their URL to see it.  But the opposite is not true: if we put `master` in the URL, it's not easy for someone to guess what version we were looking at when we wrote this to know what tag or SHA to put in the URL to see what we had in mind when we linked.

All that said: this is linking to an `examples` directory that I expect to be pretty stable, and I think it's reasonable here to link to `master` to optimize for linking folks to the current version of the example.

Either way, let me know if you want me to change it.  It's your repo and I'm happy to defer to whatever you prefer!

File: README.md
Comment (myronmarston) [reply README.md:339]: (But I'm going to wait for a reply to the other comment thread below so that if I'm changing both things I do it all at once.)

Comment (myronmarston) [reply README.md:340]: Done.

File: README.md
Comment (myronmarston) [reply README.md]: Updated to `master`.

**Review by sandwwraith — APPROVED** [review]
Thanks!

**Pull request merged**
